index.js

```diff
@@ -24,6 +26,13 @@
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
+function drinkWater(currentThirstLevel)
+{
+  console.log("Man I sure am thirsty")
+  currentThirstLevel--
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+}
```

Comment:
Good! Very nice using [camel case](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c8/CamelCase_new.svg/1200px-CamelCase_new.svg.png) for function and variable names.